README.md

```diff
@@ -4,52 +4,55 @@ The first ever Bitcoin Vault wallet.
 
 [![Playstore](https://bluewallet.io/img/play-store-badge.svg)](https://play.google.com/store/apps/details?id=io.goldwallet.wallet)
 
-
 ## BUILD & RUN IT
 
-* In your console:
+- In your console:
 
 ```
 git clone https://github.com/bitcoinvault/GoldWallet.git
 cd GoldWallet
-npm install
+yarn install
 ```
 
-* Add files for firebase:
+- Add files for firebase:
 
 For iOS:
 GoogleService-Info.plist into directory /ios/
 
 For android:
-google-services.json into directory /android/app/
+google-services.json into directory /config/prod/
+
+- Create .env file:
+
+Create .env file in /config/prod/ directory. It will be copied to root directory while building. Check the .env.example file for a template
```

Comment:
there is already .env file in the /config/prod/ so there is no need to create it. 
Can we update SENTRY DSN in it? to 
`SENTRY_DSN=https://dc67fd6d5c2949f2a6853e60eb69d899@o429172.ingest.sentry.io/5375289`

Reply:
So you want to monitor someone else's logs in our sentry?

Reply:
you are right, I guess it is the same situation as with google services file. So we should also remove our sentry dsn from env.test